playbook/app/pb_kits/playbook/pb_multi_level_select/multi_level_select.rb

```diff
@@ -16,6 +16,8 @@ class MultiLevelSelect < Playbook::KitBase
                            default: "pills"
       prop :input_name, type: Playbook::Props::String,
                         default: ""
+      prop :ultimate_children_only, type: Playbook::Props::Boolean,
```

Comment:
I see you removed this prop from the react side, I'm assuming we want to remove it from the rails side as well?